.github/workflows/ci.yml

```diff
@@ -261,10 +263,11 @@ jobs:
           node-version: latest
           check-latest: true
           cache: "yarn"
-      - name: Install and build
-        run: make -j bootstrap
+      - name: Build babel artifacts
+        run: |
+          make -j build-standalone-ci
         env:
-          BABEL_ENV: test
+          BABEL_ENV: test-legacy
```

Comment:
The `test-legacy` was introduced when we upgraded to Jest 25 which dropped Node.js 6. So we build on latest Node but targeted to a legacy Node version. However, in the Babel 8 situation, Node.js 14 is not a legacy version but a minimum supported one.

Maybe we can merge `test-legacy` with `test`, and then we differentiate node target by the `CI` environment variable. So that if a contributor runs `yarn jest`, he/she is testing against a build targeted to latest Node, which almost involves no transforms. If CI runs `yarn jest`, it is testing against the production build.